### PR TITLE
update NPU examples

### PR DESCRIPTION
**Pull request by leonardozcm**
## Description

Update verified models.

### CODE REVIEW

**Review by MeouSker77 — APPROVED** [review]

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Run LLama2 on Intel NPU
-In this directory, you will find examples on how you could apply IPEX-LLM INT4 optimizations on Llama2 models on [Intel NPUs](../../../README.md). For illustration purposes, we utilize the [meta-llama/Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) as reference Llama2 models.
+# Run Large Language Model on Intel NPU
+In this directory, you will find examples on how you could apply IPEX-LLM INT4 or INT8 optimizations on LLM models on [Intel NPUs](../../../README.md). For illustration purposes, we utilize the [meta-llama/Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) as reference Llama2 models. For more verified models please refer to the [Verification Models](#verification-models) section.
```
Comment (jason-dai) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:2]: In this directory, you will find examples on how you could apply IPEX-LLM INT4 or INT8 optimizations on LLM models on Intel NPUs. See the table blow for verified models.

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 2]
```diff
@@ -57,3 +57,15 @@ Inference time: xxxx s
...
+
+| Model      | Model Link                                                    |
+|------------|----------------------------------------------------------------|
+| Llama2 | [meta-llama/Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) |
+| Llama3 | [meta-llama/Meta-Llama-3-8B-Instruct](https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct) |
+| Chatglm3 | [THUDM/chatglm3-6b](https://huggingface.co/THUDM/chatglm3-6b) |
+| Qwen2 | [Qwen/Qwen2-7B-Instruct](https://huggingface.co/Qwen/Qwen2-7B-Instruct) |
+| MiniCPM | [openbmb/MiniCPM-2B-sft-bf16](https://huggingface.co/openbmb/MiniCPM-2B-sft-bf16) |
+| Phi-3 | [microsoft/Phi-3-mini-4k-instruct](https://huggingface.co/microsoft/Phi-3-mini-4k-instruct) |
+| Stablelm | [stabilityai/stablelm-zephyr-3b](https://huggingface.co/stabilityai/stablelm-zephyr-3b) |
```
Comment (jason-dai) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:71]: Move the table to the beginning of README

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # Run Large Language Model on Intel NPU
-In this directory, you will find examples on how you could apply IPEX-LLM INT4 or INT8 optimizations on LLM models on [Intel NPUs](../../../README.md). For illustration purposes, we utilize the [meta-llama/Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) as reference Llama2 models. For more verified models please refer to the [Verification Models](#verification-models) section.
+In this directory, you will find examples on how you could apply IPEX-LLM INT4 or INT8 optimizations on LLM models on [Intel NPUs](../../../README.md). For illustration purposes, we utilize the [meta-llama/Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) as reference Llama2 models. In this directory, you will find examples on how you could apply IPEX-LLM INT4 or INT8 optimizations on LLM models on Intel NPUs. See the table blow for verified models.
```
Comment (jason-dai) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:2]: Remove `For illustration purposes, we utilize the [meta-llama/Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) as reference Llama2 models.`

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+
+## Verification Models
```
Comment (jason-dai) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:4]: Verified Models

Comment (leonardozcm) [reply python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:4]: Updated in https://github.com/intel-analytics/ipex-llm/pull/11548

**Pull request merged**